goldens/public-api/angular/ssr/index.api.md

```diff
@@ -63,7 +63,7 @@ export interface ServerRoutePrerender extends Omit<ServerRouteCommon, 'status'>
 // @public
 export interface ServerRoutePrerenderWithParams extends Omit<ServerRoutePrerender, 'fallback'> {
     fallback?: PrerenderFallback;
-    getPrerenderParams: () => Promise<Record<string, string>[]>;
+    getPrerenderParams: () => Promise<(string[] | Record<string, string>)[]>;
```

Comment:
Another option would be to add a different option something like `getPrerenderPath` or `getPrerenderSegments`

Reply:
IMHO, reusing `getPrerenderParams` is reasonable. I think the intent behind this function is still the same whether it's filling in `:foo` or `**`. Though I'm a little worried that supporting the two in an _xor_ fashion might be confusing for developers. If we can find a reasonable way to support `:foo/**`, then using a single function LGTM.